lib/app.coffee

```diff
@@ -14,18 +14,20 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ###
 
-Raven = require('raven')
-Raven.disableConsoleAlerts()
-Raven.config require('./config').sentryDsn,
+Analytics = require('analytics.node').core
+Raven = require('analytics.node').sentryIntegration
+dsn = require('./config').sentryDsn
+release = require('../package.json').version
+ravenOptions =
+	config: dsn
+	release: release
 	captureUnhandledRejections: true
-	release: require('../package.json').version
-.install (logged, error) ->
-	console.error(error)
-	process.exit(1)
-Raven.setContext
-	extra:
-		args: process.argv
-		node_version: process.version
+	disableConsoleAlerts: true
+Analytics.addIntegration(Raven)
+Analytics.initialize 'Sentry': ravenOptions
```

Comment:
What happens with errors in setup? The previous code caught them - was that unnecessary, or is this now not catching errors if Sentry fails to start up, or is something else dealing with this elsewhere?

Reply:
I don't think we need all this setup in app.coffee after all. (unless I miss something)